.github/workflows/reusable-web-e2e.yml

```diff
@@ -116,7 +113,7 @@ jobs:
           working-directory: apps/web
           browser: "${{ env.BROWSER_PATH }}"
           record: true
-          parallel: true
+          parallel: false
```

Comment:
does that mean that the ci will be much slower? 🫣

Reply:
Unfortunately yes, but now it's still slower because it's failing every second time :(